docker-compose.yaml

```diff
@@ -24,7 +24,11 @@ services:
       - SERVERFULL_RUNTIME_STATS_DATADOG_ADDRESS=statsd:8126
       - SERVERFULL_RUNTIME_SIGNALS_INSTALLED=OS
       - SERVERFULL_RUNTIME_SIGNALS_OS_SIGNALS=15 2
-      - NEXPOSE_SITE_ASSET_PAGE_SIZE=100
+      - NEXPOSE_HOST
+      - NEXPOSE_USERNAME
+      - NEXPOSE_PASSWORD
+      - NEXPOSE_ASSETFETCHERPAGESIZE
```

Comment:
nit: this should be updated to `NEXPOSE_PAGESIZE`

Reply:
Fixed. Thanks